app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/MainSavedFragment.kt

```diff
@@ -68,15 +69,16 @@ class MainSavedFragment : BaseFragment() {
                 badgeConfig.mode
             )))
         } else {
+            Toast.makeText(context, R.string.no_configuration, Toast.LENGTH_SHORT).show()
             return DataToSend(listOf(Message(
-                Converters.convertTextToLEDHex(
-                    " ",
+                Converters.convertDrawableToLEDHex(
```

Comment:
No, Don't display a default bitmap

Reply:
try to code inside getSendData() for fragments

Reply:
If I do that, the fab animation and everything will happen which should not if we are sending nothing. I don't think it will look good if we are sending nothing and the entire fab animation still happens.

Reply:
save the returned value first, then perform the check. If it turns out to be true, proceed with the animation else dont.

Reply:
On it.